frontend/packages/kubevirt-plugin/integration-tests/v2v/rhv/v2v.rhv.configs.ts

```diff
@@ -0,0 +1,67 @@
+import { VMImportConfig, InstanceConfig } from '../../tests/utils/types';
+import {
+  IMPORT_WIZARD_CONN_NAME_PREFIX,
+  IMPORT_WIZARD_CONN_TO_NEW_INSTANCE,
+} from '../../tests/utils/consts';
+
+const {
+  V2V_INSTANCE_API_URL,
+  V2V_INSTANCE_USERNAME,
+  V2V_INSTANCE_PASSWORD,
+  V2V_INSTANCE_CA_CERT,
+  V2V_INSTANCE_CLUSTER,
+} = process.env;
+
+const newInstanceConfig: InstanceConfig = {
+  instance: IMPORT_WIZARD_CONN_TO_NEW_INSTANCE,
+  apiUrl: V2V_INSTANCE_API_URL,
+  username: V2V_INSTANCE_USERNAME,
+  password: V2V_INSTANCE_PASSWORD,
+  certificate: V2V_INSTANCE_CA_CERT,
+  cluster: V2V_INSTANCE_CLUSTER,
+  saveInstance: false,
+};
+
+const exInstanceConfig: InstanceConfig = {
+  instance: IMPORT_WIZARD_CONN_NAME_PREFIX,
+  apiUrl: V2V_INSTANCE_API_URL,
+  username: V2V_INSTANCE_USERNAME,
+  password: V2V_INSTANCE_PASSWORD,
+  certificate: V2V_INSTANCE_CA_CERT,
+  cluster: V2V_INSTANCE_CLUSTER,
+  saveInstance: false,
+};
+
+export const vmName = 'cirros-vm-for-tests';
+export const RHV = 'Red Hat Virtualization (RHV)';
+
+export const rhvVMConfig: VMImportConfig = {
```

Comment:
can we try to use the description in some test?

Reply:
Updated, added description to the basic test

Reply:
can we please check the desc is used in the final VM?